docs/config/environment.rst

```diff
@@ -118,40 +118,10 @@ General
 
       Make sure there is enough disk space available for heap dumps.
 
-.. _garbage-collection:
-
-Garbage collection
-------------------
-
-Collector
-~~~~~~~~~
-
-CrateDB uses the `G1`_ garbage collector by default.
-
-Before CrateDB 4.1 it defaulted to use the `Concurrent Mark Sweep` garbage
-collector. If you'd like to continue using CMS, you can switch setting the
-following :ref:`CRATE_JAVA_OPTS <conf-env-java-opts>`::
-
-
-  export CRATE_JAVA_OPTS="-XX:-UseG1GC -XX:+UseCMSInitiatingOccupancyOnly -XX:+UseConcMarkSweepGC"
-
-
-Logging
-~~~~~~~
```

Comment:
I removed the GC Logging section as well as this is not really related to env variables